css/style.css

```diff
@@ -118,9 +128,9 @@ nav {
 	padding: 0 1.3em;
 }
 
-nav h1 {
+/* nav h1 {
 	float: left;
-}
+} */
```

Comment:
Better delete it than risking dead code.

Reply:
Will do.